public/js/gulliver.es6.js

```diff
@@ -68,6 +68,7 @@ class Gulliver {
     this.router.addEventListener('navigate', e => {
       this.analytics.trackPageView(e.detail.url);
       this.shell.onRouteChange(e.detail.route);
+      this.offlineSupport.markAsCached(document.querySelectorAll('.offline-aware'));
```

Comment:
Can we move the string into a constant to make it easier to change the value in the future.